projects/PunktPunktStrich/index.md

```diff
@@ -0,0 +1,100 @@
+--- 
+authors: 
+  - "Malte Völkner"
+authors-url: 
+  - "https://github.com/MalteVoelkner"
+layout: project
+title: PunktPunktStrich / ..-
+summary: A introduction into computer generated art
+---
+
+![Title picture](https://github.com/MalteVoelkner/PunktPunktStrich/blob/master/splash.png)
+
+## Description
+The following project shows our introduction into the most important thing of computer generated art: algorithms. We took advantage of basic, analog algorithms to get a better understanding on how algorithms work and how to improve them throughout the process of creating art.
+
+The assignment was part of the course [Gestalten in Code – Die Grundlagen Generativer Gestaltung](https://incom.org/workspace/6886) held by [Fabian Morón Zirfas](http://fabianmoronzirfas.me/) at the [University of Applied Sciences Potsdam](https://www.fh-potsdam.de/) in winter 2016/2017.
+
+## Assignment
+The native assignment was to create a simple, analog algorithm. The algorithm was supposed to be executed by fellow students and to be improved based on their feedback. Further conditions stated that the algorithm needs to be doable on paper, as well as taking a maximum of five minutes to be executed. An additional term was a strict definition of the used hardware (*for example the type of paper used*).
+
+## Inspiration
+I was inspired to work with dots and lines by Morse code and its alphabet. Morse code is a method of transmitting text as a series of on- or off-signals (*with the best-known being “SOS”: ••• — — — •••*). Morse code inspired me by being a great comparison to analog and digital algorithms: a human input changes its form into a computer-readable input while keeping the same message. Written code has a fascinating look and got me to persuade it as my perception of computer generated art.
+
+
+## 1ˢᵗ Iteration
+These are the first instructions that were given out for the first few try-outs:
+
+```
+Black lines in a squared grid only.
+Solely horizontal lines, only from dot to dot.
+Each row has to begin with a line with the length “x”.
+All following lines have to be either half- or twice as long as the previous line.
+Between every row needs to be a distance of “x”.
+
+You have five minutes to complete this task.
+```
+
+The executors were all given a DIN A5-sized paper (148 by 210mm, squared grid) and the same pen, a black marker with a medium thickness. Besides the abovementioned instructions no other information were given. These were the first results I got back:
+
+![First Version](https://github.com/MalteVoelkner/PunktPunktStrich/blob/master/assets/images/First_Version.png)
```

Comment:
relative paths please. Also the filenames should be lowercase  

    ![](./assets/images/first_version.png)